app/controllers/api/v1/customers_controller.rb

```diff
@@ -70,6 +99,10 @@ def visible_customers
         )
       end
 
+      def editable_enterprises
+        OpenFoodNetwork::Permissions.new(current_api_user).editable_enterprises.select(:id)
+      end
```

Comment:
Why did you move this method?

Reply:
It made more sense for the order of method calls and to have params at bottom, guess it wasnt worth it for rewriting history.